### DISCUSSION TITLE
4.1.2 release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1635] [General]
All devs,

Since I'll be out of work for 3 weeks soon (my first son is coming next week!), I'm planning to release a 4.1.2 version. Current release notes are:
https://github.com/sepinf-inc/IPED/blob/4.1.x/ReleaseNotes.txt

Should we include something else?

PS: I'm in doubt about #1608 since it has many changes and I'm not sure if I will have time to test it properly...

**Comment by paulobreim** [comment]
It is ok for me.
